best-practices/add-rolebinding/add-rolebinding.yaml

```diff
@@ -44,4 +45,4 @@ spec:
             name: admin
           subjects:
             - kind: User
-              name: "{{request.userInfo.username}}"
\ No newline at end of file
+              name: "{{request.userInfo.username}}"
```

Comment:
If we're not materially changing code, we should not be submitting these sort of changes to a public repo.  It just leads to more noise when we merge from public upstream. Can this change be reverted?